mobile_marketplace/mobiles/tasks.py

```diff
@@ -0,0 +1,16 @@
+from django.utils import timezone
+from datetime import timedelta
+from mobile_marketplace.mobiles.models import Mobile
+from celery import shared_task
+
+
+@shared_task
+def deleting_mobiles_after_14_days():
+    fourteen_days_ago = timezone.now() - timedelta(days=14)
```

Comment:
No Need to create separate variable for this. Because it's not resuable. Also, not affecting the readibility.